ui/apps/platform/src/Containers/Dashboard/PatternFly/DashboardPage.css

```diff
@@ -0,0 +1,13 @@
+/*
+These styles are to unset the conflicting rules from selector:
+#main-page-container > div > :not(.pf-c-page__main-section)
```

Comment:
The referenced rule from `style.css` essentially overrides anything added to the main page component that isn't directly wrapped in a PF `<PageSection>`. I believe these are global styles for older Tailwind pages, but they also overwrite defaults on newer pages depending on the component used.

We might be able to change (or append) to that rule to make sure it is ignored for all PF pages by changing the selector to reference a CSS class from PF's `<Page>` component, and ensuring all new pages are wrapped with it. Probably better to discuss in a team meeting though, as it could have some wide reaching impact in the app.

Reply:
This seems like a reasonable temporary work-around to keep forward momentum.

The need to diverge from general principle that content on PatternFly page is contained in one or more `PageSection` elements confirmed the need to follow up on question about recent trend in its use.

Looking at it in a different way, what are pro and con to render `SummaryCounts` element below the main heading of the page? Putting it above the heading was a temporary compromise because it had been on the classic top bar. In PatternFly it looks out of place above the **Dashboard** heading.

Reply:
Good question - I kept it like this because that's how it was in the mocks, and how it has been in the past. We could run the question by UX though and see if it makes sense to move it.

The main con I see is that it will be a little different than what users expect, and the impression I got from Slack convos is that users _really_ like this information as-is.

Reply:
In case it needs to stay at the top, then what do you think about custom style if needed for a `PageSection` to contain `SummaryCounts` element, instead of exception to page element hierarchy?

Can you double check whether or not my memory and understanding is correct: idiomatic PatternFly for `Page` element is main content in one or more `PageSection` elements?

Reply:
Hmm it is a little ambiguous in the [PatternFly docs](https://www.patternfly.org/v4/components/page/accessibility):
> Page layouts are defined using page sections, such as the header, body, and footer of a page. 
but then also
> A page can be combined with many other components
, but I'd say leaning towards using <PageSection> in general.

I went with a non-section element due to the semantic issue you brought up the other day, but for consistency and avoiding this override I can convert to <PageSection>. From what I can see in our app, the vast majority (all?) of pages used either PageSection or Divider as the child elements of <Page>.

I also opened an [issue with PF](https://github.com/patternfly/patternfly-react/issues/7419) regarding switching out the element, so if they are in agreement we might be able to change these out with <div>s in the future.